infrastructure/src/main/java/org/corfudb/infrastructure/NettyServerRouter.java

```diff
@@ -168,20 +131,21 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) {
             // pipeline.
             CorfuMsg m = ((CorfuMsg) msg);
             // We get the handler for this message from the map
-            AbstractServer handler = handlerMap.get(m.getMsgType());
-            if (handler == null) {
+            AbstractServer server = handlerMap.get(m.getMsgType());
```

Comment:
Isn't handler a more precise description of what an AbstractServer does? otherwise maybe the handlerMap should also changed to serverMap.

Reply:
Changed back.